integration_tests/pages/appointments/appointment/appointmentDetailsPage.ts

```diff
@@ -20,9 +20,8 @@ export default class AppointmentDetailsPage extends Page {
       .parents('[data-qa=prisoner-summary]')
       .find('[data-qa=prisoner-number]')
       .contains(number)
-      .parents('[data-qa=prisoner-summary]')
-      .find('[data-qa=prisoner-cell-location]')
-      .contains(cellLocation)
+
+    cy.get('[data-qa=prisoner-cell-location]').contains(cellLocation)
```

Comment:
I wonder if this should be in its own method now the cell location has been moved out of the prisoner summary and into it's own column?

Could also be useful to verify that the cell location found is associated with the correct prisoner row.